File: HeliumExample/src/main/java/com/chartboost/heliumsdk/sampleapp/SplashActivity.java
package com.chartboost.heliumsdk.sampleapp;

import android.app.Activity;
import android.content.Intent;
import android.os.Bundle;
import android.widget.Toast;

import com.chartboost.heliumsdk.HeliumSdk;

import org.jetbrains.annotations.Nullable;


public class SplashActivity extends Activity {


    @Override
    public void onCreate(@Nullable Bundle savedInstanceState) {

        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_splash);

        HeliumSdk.start(this, getString(R.string.appID), getString(R.string.appSignature), error -> {
            if (error != null) {
                runOnUiThread(() -> {
                    String errorMessage = "Helium SDK failed to initialize. Reason: " + error.getMessage();
                    Toast.makeText(SplashActivity.this, errorMessage, Toast.LENGTH_LONG).show();
                });
            } else {
                Intent mainActivityIntent = new Intent(SplashActivity.this, MainActivity.class);
                SplashActivity.this.startActivity(mainActivityIntent);
                finish();
            }
        });

        HeliumSdk.setDebugMode(false);// Only for debug builds!
    }
}

File: HeliumExample/src/main/java/com/chartboost/heliumsdk/sampleapp/SplashActivity.kt
package com.chartboost.heliumsdk.sampleapp

import android.content.Intent
import android.os.Bundle
import android.widget.Toast
import androidx.appcompat.app.AppCompatActivity
import com.chartboost.heliumsdk.HeliumSdk

class SplashActivity : AppCompatActivity() {
    public override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_splash)
        HeliumSdk.start(
            this,
            getString(R.string.appID),
            getString(R.string.appSignature)
        ) { error: Error? ->
            if (error != null) {
                runOnUiThread {
                    val errorMessage = "Helium SDK failed to initialize. Reason: " + error.message
                    Toast.makeText(this@SplashActivity, errorMessage, Toast.LENGTH_LONG).show()
                }
            } else {
                val mainActivityIntent = Intent(this@SplashActivity, MainActivity::class.java)
                this@SplashActivity.startActivity(mainActivityIntent)
                finish()
            }
        }
        HeliumSdk.setDebugMode(false) // Only for debug builds!
    }
}